packages/utils/src/lib/reports/sorting.ts

```diff
@@ -60,7 +60,7 @@ export function sortReport(report: ScoredReport): ScoredReport {
       ...audit,
       details: {
         ...audit.details,
-        issues: audit?.details?.issues.slice().sort(compareIssues) || [],
+        issues: [...(audit?.details?.issues ?? [])].sort(compareIssues),
```

Comment:
I'm not sure about this, the sorting will add en empty `issues` array even when one wasn't produced. I see it was also always creating a `details` object even if there wasn't one originally. Semantically there may not be much of a difference (although `issues: []` to me indicates that this audit _could_ produce issues), but it could bloat the JSON file size needlessly, and I wouldn't expect sorting to have this effect.

I'd suggest a softer approach. If `issues` aren't defined, just leave the original object unchanged:

```ts
audit =>
  audit.details?.issues
    ? { ...audit, details: { ...audit.details, issues: [...audit.details.issues].sort(compareIssues) } }
    : audit
```